src/components/ui-shell/side-nav.ts

```diff
@@ -81,6 +81,9 @@ class BXSideNav extends HostListenerMixin(LitElement) {
    */
   private _handleButtonToggle(event: CustomEvent) {
     this.expanded = event.detail.active;
+    if (this.expanded) {
+      (this.querySelector((this.constructor as typeof BXSideNav).selectorNavItems) as HTMLElement).focus();
```

Comment:
My apologies, forgot to cover a condition where the focusable element is not found:

```suggestion
      (this.querySelector((this.constructor as typeof BXSideNav).selectorNavItems) as HTMLElement)?.focus();
```